### PR TITLE
Change the indicator resource endpoint from the old version of Summar…

### PR DESCRIPTION
**Pull request by MorningSunKing**
…y API to metrics resource

### CODE REVIEW

**Comment by mengjiao-liu** [comment]
Preview page: https://deploy-preview-41707--kubernetes-io-main-staging.netlify.app/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/

File: content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ P[pod data] -.- C
 end
 L[API<br>server]
 W[HPA]
-C ---->|Summary<br>API| A -->|metrics<br>API| L --> W
+C ---->|metrics<br>resource| A -->|metrics<br>API| L --> W
```
Comment (mengjiao-liu) [inline content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md:53]:
```suggestion
C ---->|metrics resource<br>API| A -->|metrics<br>API| L --> W
```
It's easier to understand.


Have any suggestions？@sftim  @tengqm

Comment (pacoxu) [reply content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md:53]: Agree with the suggested change. 
- `Metrics resource api` is clearer than `metrics resource` for `/metrics/resource` API.

File: content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md
Comment (sftim) [inline content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md:1]: If making this change, you must also change:
```markdown
* [Summary API](#summary-api-source): API provided by the kubelet for discovering and retrieving
  per-node summarized stats available through the `/stats` endpoint.
```

**Review by sftim — COMMENTED** [review]
/lgtm cancel

This PR only makes part of the necessary changes. We should amend it before we merge.

File: content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md:53]:
```suggestion
C ---->|node level<br>resource metrics| A -->|metrics<br>API| L --> W
```

File: content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md
Comment (MorningSunKing) [reply content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md:1]: Done.

Comment (MorningSunKing) [reply content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md:1]: > If making this change, you must also change:
> 
> ```
> * [Summary API](#summary-api-source): API provided by the kubelet for discovering and retrieving
>   per-node summarized stats available through the `/stats` endpoint.
> ```
Done.

**Pull request merged**